app/connectors/test/TestOnlySecureMessagingConnector.scala

```diff
@@ -81,4 +88,25 @@ class TestOnlySecureMessagingConnector @Inject()(
       .withBody(jsonPayload)
       .execute[HttpResponse]
   }
-}
\ No newline at end of file
+
+  def getMessages(
```

Comment:
since this is used within the actual service. Can we move this away from testOnly as this isn't a testOnly method